portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/siteStatus-snippet.vm

```diff
@@ -8,7 +8,25 @@
 
     <div id="siteStatus" class="Mrphs-siteStatus Mrphs-messagePanel__information is-unpublished">
         <h4>${rloader.sit_unpublished}</h4>
-        <button onclick="publishSite('${sitePages.siteId}');" class="Mrphs-button--active">${rloader.sit_publish_now}</button>
+        <button class="Mrphs-button--active" data-toggle="modal" data-target="#publishSiteConfirmModal">${rloader.sit_publish_now}</button>
+    </div>
+
+    <div id="publishSiteConfirmModal" class="modal fade" role="dialog" tabindex="-1" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal">&times;</button>
+                    <h4 class="modal-title">${rloader.sit_unpublished}</h4>
+                </div>
+                <div class="modal-body">
+                    <p>${rloader.sit_publish_confirmation}</p>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-primary" data-dismiss="modal" onclick="publishSite('${sitePages.siteId}');">${rloader.sit_publish_now}</button>
```

Comment:
@fostersdesign what are the classes for these buttons? these classes should be correct but doesn't work for me.

Reply:
.btn-primary should be correct. Is it being overridden by a library?

Reply:
For some reason, btn-primary doesn't work for me, this PR should be correct.

Reply:
I mocked up the situation and noticed that `button:not(.fc-button)` is causing `button.btn-primary` to not have enough specificity to make it look like the primary button. 

`button:not(.fc-button)` is such a broad rule that was added for the FullCalendar work done in the Lessons, I believe. I have noticed this issue in other cases that I've been working on in the recent past and have been thinking about removing it in favour of addressing the buttons in FullCalendar separately. 

So, I'm okay getting your change here into Sakai, @mpellicer. Then we can fix the `button:not(.fc-button)` separately.

Reply:
good